src/transactions/feed/TransactionFeed.tsx

```diff
@@ -84,6 +84,8 @@ function TransactionFeed() {
         return <NftFeedItem key={tx.transactionHash} transaction={tx} />
       case 'TokenApproval':
         return <TokenApprovalFeedItem key={tx.transactionHash} transaction={tx} />
+      case 'JumpstartDeposit':
```

Comment:
nit: what if we list this case right below `TokenTransferV3`?
```typescript
case 'TokenTransferV3':
case 'JumpstartDeposit':
  return <TransferFeedItem key={tx.transactionHash} transfer={tx} />
```

Reply:
Yes, sorry! At first I was considering using a different component...
Thanks!